Plugins/Public/alley/PlayerRestrictions.cpp

```diff
@@ -768,31 +768,34 @@ bool UserCmd_Process(uint iClientID, const wstring &wscCmd)
 }
 
 #define IS_CMD(a) !wscCmd.compare(L##a)
-
+#define RIGHT_CHECK(a) if(!(cmds->rights & a)) { cmds->Print(L"ERR No permission\n"); return true; }
 bool ExecuteCommandString_Callback(CCmds* cmds, const wstring &wscCmd)
 {
 	returncode = DEFAULT_RETURNCODE;
 
 	if (IS_CMD("showrestrictions"))
 	{
 		returncode = SKIPPLUGINS_NOFUNCTIONCALL;
+
+		RIGHT_CHECK(RIGHT_SUPERADMIN)
+
 		return true;
 	}
 	else if (IS_CMD("generateid"))
 	{
 		returncode = SKIPPLUGINS_NOFUNCTIONCALL;
+
+		RIGHT_CHECK(RIGHT_SUPERADMIN)
```

Comment:
this is actually harmless and should probably not even be an admincmd - actually I would question why it's an FLHook command at all, and not some out-of-game utility. But this is the existing check so okay 👍